### PR TITLE
Update 0407.lang

### PR DESCRIPTION
**Pull request by speedy-pc**
- Add RTL support
- Add enhanced accessibility
- improved translation

### CODE REVIEW

**Comment by micwoj92** [comment]
@botflakes, @JimBim89, @schmelto,  @htcfreek

**Review by schmelto — APPROVED** [review]
Lgtm

File: Langs/0407.lang [hunk 1]
```diff
@@ -9,18 +10,20 @@ Medium=10
 Small=8.5
 Name=Arial
 [Strings]
+Enable enhanced accessibility?=Erweiterte Barrierefreiheit aktivieren?
+This new Enhanced Accessibility functionality is designed for the visually impaired, in which most of the program interface can be used by voice and keyboard shortcuts. Activate?=Die neue erweiterte Barrierefreiheit-Funktionalität wurde für Sehbehinderte entwickelt, bei denen der größte Teil der Programmoberfläche durch Sprach- und Tastaturkürzel verwendet werden kann. Soll diese aktiviert werden?
```
Comment (htcfreek) [inline Langs/0407.lang:14]:
```suggestion
This new Enhanced Accessibility functionality is designed for the visually impaired, in which most of the program interface can be used by voice and keyboard shortcuts. Activate?=Die neue Funktionalität für erweiterte Barrierefreiheit wurde für Sehbehinderte entwickelt, wobei der größte Teil der Programmoberfläche durch Sprach- und Tastaturkürzel kontrolliert werden kann. Soll diese aktiviert werden? 
```

File: Langs/0407.lang [hunk 2]
```diff
@@ -69,14 +72,14 @@ Done=Fertig
...
-Boot=Eine Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. "Legacy" bedeutet, dass sich ihr System im traditionellen Startmodus befindet. Dies kann in den BIOS/UEFI-Einstellungen korrigiert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
-CPU Name=Der Prozessor, welcher in Ihrem Computer verbaut ist. Die Liste der zu Windows 11 kompatiblen Prozessoren kann sich noch ändern. Bei einer Inkompatibilität muss der Prozessor ersetzt werden. Achtung: Dies ist bei vielen Laptops nicht möglich.
-CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
-CPU Speed=Die Geschwindigkeit, mit der Ihre CPU die Aufgaben abarbeitet. Bei einer Inkompatibilität muss die CPU ersetzt werden. Achtung: Dies ist bei vielen Laptops nicht möglich.
+Architecture=Sollten Sie ein 32-Bit-Betriebssystem nutzen, erfordert dies eine 64-Bit-Neuinstallation von Windows 11. Ihre Daten müssen vorab gesichert werden. Eine 32-Bit-CPU erfordert einen Wechsel auf eine 64-Bit-CPU.
```
Comment (htcfreek) [inline Langs/0407.lang:75]:
```suggestion
Architecture=Sollten Sie ein 32-Bit-Betriebssystem einsetzen, erfordert dies eine 64-Bit-Neuinstallation von Windows 11. Ihre Daten müssen vorab gesichert werden. Eine 32-Bit-CPU erfordert einen Wechsel auf eine neue 64-Bit-CPU.
```

Sounds better.

**Review by htcfreek — CHANGES_REQUESTED** [review]

File: Langs/0407.lang [hunk 3]
```diff
@@ -69,14 +72,14 @@ Done=Fertig
...
+Boot=Die Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. Sollten Sie "Legacy" (traditioneller Startmodus) nutzen, kann dies oft in den BIOS/UEFI-Einstellungen umgestellt werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
```
Comment (htcfreek) [inline Langs/0407.lang:76]: I read somewhere that manufacturer provide new fw updates to enable uefi and tpm on the devices.

Maybe we can add a hint on this in general and for all languages.

_cc: @rcmaehl, @micwoj92 _

File: Langs/0407.lang [hunk 4]
```diff
@@ -69,14 +72,14 @@ Done=Fertig
...
+RAM=Die Größe des in Ihrem Computer installierten Arbeitsspeichers. Windows 11 erfordert mindestens 4 GB RAM. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
 Secure Boot=Secure Boot (sicherer Start) ist eine Funktion innerhalb des UEFI-BIOS. Für Windows 11 muss diese Funktion vorhanden sein. Wenn Secure Boot nicht erkannt wird, kann dies bei neueren PCs oder Laptops im UEFI-BIOS aktiviert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
-Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Windows 11 erfordert mindestens 64 GB Speicherplatz. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
 TPM=Trusted Platform Module (TPM) ist ein Sicherheitsmodul des Computers, das von Windows 11 benötigt wird. Nicht immer ist ein separates TPM-Modul notwendig. Viele moderne CPUs enthalten diese Funktion. Die Aktivierung von TPM ist oft in den UEFI-BIOS-Einstellungen möglich. Im BIOS nennt sich diese Funktion "fTPM" (AMD) oder "PTT" (Intel). Schlagen Sie im Handbuch Ihres Motherboards nach.
```
Comment (htcfreek) [inline Langs/0407.lang:85]: I read somewhere that manufacturer provide new fw updates to enable uefi and tpm on the devices.

Maybe we can add a hint on this in general and for all languages.

_cc: @rcmaehl, @micwoj92 _

**Review by htcfreek — COMMENTED** [review]
Two suggestions for additional information. @rcmaehl and @micwoj92 please state your opinion on this.

File: Langs/0407.lang (hunk 4 above)
Comment (speedy-pc) [reply Langs/0407.lang:85]: That's correct. Many manufacturers want to offer a BIOS update to generally activate TPM. 
In addition, some OEMs offer a TPM firmware update from 1.2 to 2.0. 

https://www.neowin.net/news/asus-begins-rolling-out-windows-11-compatible-firmware/
https://support.microsoft.com/en-us/windows/update-your-security-processor-tpm-firmware-94205cbc-a492-8d79-cc55-1ecd6b0a8022
https://www.danielengberg.com/hp-tpm-firmware-version-upgrade-from-1-2-to-2-0-tpm-configuration-utility/

**Pull request merged**